content/components/recipes/details/arguments/MultiPreferenceExperiment.js

```diff
@@ -85,6 +90,29 @@ export default function MultiPreferenceExperiment({ data }) {
                       <code>{branch.ratio}</code>
                     </div>
                   </div>
+                  <div className="flex-basis-0 flex-grow-1">
+                    <div className="d-flex align-items-center">
+                      <div className="pr-2 font-weight-bold">
+                        Testing clientId
+                        <HelpIcon>
+                          Setting the preference{" "}
```

Comment:
nit: i found this a little confusing `{" "}`. It should probably be `&nbsp;`

Reply:
This is how prettier formats text when it wraps JSX lines. I tested and Prettier will leave `&nbsp;` if I put it there, but I think the automatic thing is easier.

Reply:
oh right, this is all coming back to me.